firmware/controllers/algo/rusefi_types.h

```diff
@@ -23,6 +23,10 @@ typedef unsigned int time_t;
 #include "rusefi_enums.h"
 #include "firing_order.h"
 
+#if __cplusplus
```

Comment:
do we still actually include this file from a `.c` file?

Reply:
yeah, it gets included in the precompiled headers which are used by main.c and os_util.c, among others.

Reply:
![image](https://user-images.githubusercontent.com/568254/140593775-3195857e-642b-4c5f-b8a3-418e02228dea.png)

Reply:
oh, but there are other headers that include `rusefi_types.h` that are included from `.c`